src/exo/libs/malloc.c

```diff
@@ -1,12 +1,13 @@
-// clang-format off
-// these are filled in by Python's str.format()
-#define HEAP_SIZE {heap_size}
-// clang-format on
+#include "malloc.h"
```

Comment:
Although it isn't reserved by the C standard, `malloc.h` is a header name used by many C standard library implementations, anyway, [including Apple's](https://opensource.apple.com/source/Libc/Libc-825.26/include/malloc/malloc.h.auto.html). Please rename it to something like `custom_malloc.h/c` to avoid conflicts.